main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.scala

```diff
@@ -34,6 +34,9 @@ object PrimitiveEffects {
   /** The path to the method effects. */
   private val MethodEffsPath = "/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json"
 
+  /** The path to the package effects. */
+  private val PackageEffsPath = "/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Packages.json"
```

Comment:
Could we move this above ClassEffsPaths, so the order is packages, classes, constructors, methods